Extension/src/LanguageServer/settings.ts

```diff
@@ -99,7 +118,15 @@ export class OtherSettings {
     public get searchExclude(): vscode.WorkspaceConfiguration { return vscode.workspace.getConfiguration("search", this.resource).get("exclude"); }
     public get settingsEditor(): string { return vscode.workspace.getConfiguration("workbench.settings").get<string>("editor"); }
 
+    public get colorTheme(): string { return vscode.workspace.getConfiguration("workbench", this.resource).get<string>("colorTheme"); }
+
+    public getCustomColorToken(colorTokenName: string): string { return vscode.workspace.getConfiguration("editor.tokenColorCustomizations", this.resource).get<string>(colorTokenName); }
```

Comment:
There are a bunch of warnings in the console regarding accessing the tokenColorCustomizations:

`[ms-vscode.cpptools] Accessing a window scoped configuration for a resource is not expected. To associate 'editor.tokenColorCustomizations' to a resource, define its scope to 'resource' in configuration contributions in 'package.json'.`

I think you need to pass `null` for the resource in these cases.

Reply:
(or just don't pass a second parameter)